Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM amazon/aws-cli:amd64
+
```

Comment:
Do we need a Dockerfile for each arch?

Reply:
This plugin is being built mainly for amd64 arch